examples/feedback/metrics-server/config/multiclass-deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         - name: "PREDICTIVE_UNIT_ID"
           value: "classifier"
         - name: "PREDICTIVE_UNIT_IMAGE"
-          value: "alibi-detect-server:1.7.0-dev"
+          value: "seldonio/alibi-detect-server:1.12.0-dev"
```

Comment:
Was this one teh one that was added to the release.py

Reply:
I still need to add it to release.py (or check if maybe with `seldonio/` it is caught now)

Reply:
But yeah, that was the one we talked about  that it stayed on 1.7.0-dev